packages/ng-primitives/accordion/src/accordion-content/accordion-content.ts

```diff
@@ -56,6 +63,34 @@ export class NgpAccordionContent<T> {
     })
       .pipe(debounceTime(0), safeTakeUntilDestroyed())
       .subscribe(() => this.updateDimensions());
+
+    /**
+     * Compatibility with older browsers that do not support the beforematch event
+     * The hidden attribute must be removed.
+     */
+    afterRenderEffect({
+      write: () => {
+        if (!this.supportsBeforeMatch()) return;
+
+        if (this.accordionItem().open()) {
+          this.elementRef.nativeElement.removeAttribute('hidden');
+        } else {
+          this.elementRef.nativeElement.setAttribute('hidden', 'until-found');
```

Comment:
I notice that whenever we collapse the accordion (if it's animated) the content disappears immediately - I wonder is there any way, to only have the content hidden whenever the height is 0, perhaps a resize observer could be useful for this?

Let me know if this makes sense.